### PR TITLE
T-13 Add enums in code

### PR DESCRIPTION
**Pull request by goremukin**

### CODE REVIEW

**Comment by goremukin** [comment]
[T-13 Add enums in code](https://trello.com/c/23A5KQVI/13-t-13-add-enums-in-code)

File: PortfolioKeeper/Enums/AssetTypeValue.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace PortfolioKeeper.Enums
+{
+    public enum AssetTypeValue
+    {
+        Realty = 1,
```
Comment (EdwOK) [inline PortfolioKeeper/Enums/AssetTypeValue.cs]: sort all or not only first

File: PortfolioKeeper/Enums/BankValue.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace PortfolioKeeper.Enums
+{
+    public enum BankValue
+    {
+        Belgazprombank = 1,
```
Comment (EdwOK) [inline PortfolioKeeper/Enums/BankValue.cs]: sort all or not only first

File: PortfolioKeeper/Enums/CurrencyValue.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿namespace PortfolioKeeper.Enums
+{
+    public enum CurrencyValue
+    {
+        BYN = 1,
```
Comment (EdwOK) [inline PortfolioKeeper/Enums/CurrencyValue.cs]: sort all or not only first

File: PortfolioKeeper/Enums/MetalTypeValue.cs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿namespace PortfolioKeeper.Enums
+{
+    public enum MetalTypeValue
+    {
+        Gold = 1,
```
Comment (EdwOK) [inline PortfolioKeeper/Enums/MetalTypeValue.cs]: sort all or not only first

**Pull request merged**